src/CommandOptions.mjs

```diff
@@ -21,61 +21,61 @@ export const makeConfig = (argv) => {
+  You can specify the extraction level via the '--depth' option.
 
   If you have any custom elements inserted under <div d=#content>
-  in the source single html, asciidoctor-chunker ignores it by
-  default. If you want them to be included into the chunked html,
-  set the option --no-strictMode. The element will be copied to
-  every chunked page.
-
-  By default asciidoctor-chunker.css is included in the
-  output directory. It provides the non-opinionated page
-  navigation at the bottom of every chunked page. You can
-  override this by giving a comma separated list of paths
-  to your custom css files. They are copied into the output
-  directory so the paths must be accessible by
-  asciidoctor-chunker.
+  in the single-source html, asciidoctor-chunker ignores them by
+  default.  If you want them to be included into the chunked html,
+  set the '--no-strictMode' option.  The elements will be copied
+  to every chunked page.
+
+  By default 'asciidoctor-chunker.css' is included in the
+  output directory.  It provides the non-opinionated page
+  navigation bar at the bottom of every chunked page.
+  You can override this by giving a comma-separated list
+  of paths to your custom css files.  They will be copied
+  into the output directory, so their paths must be accessible
+  to asciidoctor-chunker.
 
 The Depth Specifier:
-  You can list the multiple settings by connecting each
-  specifier with a comma. Each specifier is consisted of
-  either a single number or a collon separated with two
-  numbers.
-
-  The single number sets the default level of extraction.
-  Number 1 is the application's default and it extracts the
-  chapter level. Number 2 for section extraction, 3 for
-  subsection, and so on to 6 which is the maximum section
+  You can list multiple settings by separating each specifier
+  with a comma.  Each specifier consists of either a single
+  number or two numbers separated by a colon.
+
+  A single number sets the default level of extraction.
+  Number 1 is the application default and it extracts the
+  chapter level.  Number 2 is for section extraction, 3 for
+  subsection, and so on, up to 6 which is the maximum section
   level of Asciidoctor.
 
-  The list of collon separated numbers, chap:level, can
-  change the extraction depth for specific chapters,
-  so 3:2 means chapter 3 is extracted down to 2 levels (i.e.
-  section level). You can use a hyphen to specify the range
-  of chapters to set as chapFrom-chapTo:level, so 1-3:5 means
-  chapter 1 through 5 should be extracted with the depth
+  The list of colon-separated numbers, chap:level, can
+  change the extraction depth for specific chapters.
+  So 3:2 means that chapter 3 is extracted down to 2 levels
+  (i.e. section level).  You can use a hyphen to specify the
+  range of chapters to set as chapFrom-chapTo:level, so 1-3:5
+  means chapter 1 through 5 should be extracted with the depth
```

Comment:
I like to keep the original except the word `collon-separated` in the first line as you proposed.